tests/storage/glusterfs/conftest.py

```diff
@@ -0,0 +1,128 @@
+import pytest
+
+def _setup_host_with_glusterfs(host):
+    assert not host.file_exists('/usr/sbin/glusterd'), \
+        "glusterfs-server must not be installed on the host at the beginning of the tests"
+    host.yum_install(['glusterfs-server'], save_state=True)
+    host.ssh(['systemctl', 'enable', '--now', 'glusterd.service'])
+
+    for h in host.pool.hosts:
+        hostname_or_ip = h.hostname_or_ip
+        if hostname_or_ip != host.hostname_or_ip:
+            host.ssh(['iptables', '-I', 'INPUT', '-p', 'all', '-s', hostname_or_ip, '-j', 'ACCEPT'])
```

Comment:
no idea, need to see the driver's code

Reply:
The driver's code or rather probably gluster docs

Reply:
so there are several ports that need to be opened from my understanding, so it's simpler to keep all, and since the rules are removed at the end and its for testing purposes I think its acceptable.

Reply:
Opening just the required ports would still have two advantages:
- serve as documentation
- detect if (though unlikely) future glusterfs updates require more ports to be opened